src/wallet/rpc/wallet.cpp

```diff
@@ -234,8 +234,21 @@ static RPCHelpMan loadwallet()
                     }
                 },
                 RPCExamples{
-                    HelpExampleCli("loadwallet", "\"test.dat\"")
-            + HelpExampleRpc("loadwallet", "\"test.dat\"")
+                    "\nLoad wallet with files under wallets/foo/:\n"
+                    + HelpExampleCli("loadwallet", "\"foo\"")
+                    + HelpExampleRpc("loadwallet", "\"foo\"")
+                    + "\nLoad wallet with files under wallets/foo/specialWallet/:\n"
+                    + HelpExampleCli("loadwallet", "\"foo/specialWallet/\"")
+                    + HelpExampleRpc("loadwallet", "\"foo/specialWallet/\"")
+                    + "\nLoad wallet using absolute path (Unix):\n"
+                    + HelpExampleCli("loadwallet", "\"/path/to/specialWallet/\"")
+                    + HelpExampleRpc("loadwallet", "\"/path/to/specialWallet/\"")
+                    + "\nLoad wallet using absolute path (Windows):\n"
+                    + HelpExampleCli("loadwallet", "\"C:\\Users\\myusername\\specialWallet\\\"")
+                    + HelpExampleRpc("loadwallet", "\"C:\\Users\\myusername\\specialWallet\\\"")
+                    + "\nLoad legacy wallet.dat file directly under the wallets/ directory:\n"
+                    + HelpExampleCli("loadwallet", "\"legacy.dat\"")
+                    + HelpExampleRpc("loadwallet", "\"legacy.dat\"")
```

Comment:
Could reduce it down to only one `HelpExampleRpc` if one would want to reduce the number of lines.
```suggestion
                    + "\nLoad wallet with files under wallets/foo/ via RPC:\n"
                    + HelpExampleRpc("loadwallet", "\"foo\"")
                    + "\nLoad wallet with files under wallets/foo/specialWallet/:\n"
                    + HelpExampleCli("loadwallet", "\"foo/specialWallet/\"")
                    + "\nLoad wallet using absolute path (Unix):\n"
                    + HelpExampleCli("loadwallet", "\"/path/to/specialWallet/\"")
                    + "\nLoad wallet using absolute path (Windows):\n"
                    + HelpExampleCli("loadwallet", "\"C:\\Users\\myusername\\specialWallet\\\"")
                    + "\nLoad legacy wallet.dat file directly under the wallets/ directory:\n"
                    + HelpExampleCli("loadwallet", "\"legacy.dat\"")
```

Reply:
Thank you, let's see if others want this change too. I think it could be acceptable to have more example lines.

Reply:
Yeah, I could accept either way. You have shown good persistence with this issue @am-sq!

Reply:
Hi @hodlinator do you know if there is something I can do to get this merged?

Reply:
Don't know of a good way unfortunately. Maybe one of the other reviewers will react soon from our comment activity today.

Did a quick search for `HelpExampleRpc` now and there are a few other RPCs which have quite a few examples.